src/components/Guilds/Filter/index.tsx

```diff
@@ -24,15 +26,37 @@ const FilterRow = styled.div`
   }
 `;
 
+const ButtonContainer = styled(Flex)`
+  flex-direction: Row;
+`;
+
 export const Filter = () => {
   const [viewFilter, setViewFilter] = useState(false);
+  const [createProposal, setCreateProposal] = useState(true);
   const { totalFilters } = useFilter();
+
+  const history = useHistory();
+  const location = useLocation();
+
   return (
     <FilterContainer>
       <FilterRow>
         <InputText placeholder="Proposal title" />
-        {isDesktop && <FilterMenu />}
-        {isMobile && (
+        {createProposal && (
+          <ButtonContainer>
+            <Button onClick={() => setCreateProposal(false)}>
+              Proposal state
+            </Button>
+            <Button
+              variant="secondary"
+              onClick={() => history.push(location.pathname + '/ProposalType')}

```

Comment:
Maybe we can add`proposalType` to be consistent with the rest of the routes on the app.
```suggestion
              onClick={() => history.push(location.pathname + '/proposalType')}

```

Reply:
Good catch. I will change this